rs/tests/driver/src/k8s/tnet.rs

```diff
@@ -360,15 +360,27 @@ impl TNet {
             }
         );
 
+        // We make reservation for 42 vcpus only if VM uses 64 vcpus because there are
+        // already other k8s resources having resource requests that prevents reservation to succeeds.
+        // Note that VM still gets 64 vcpus.
+        let vcpus = if vm_req.vcpus.to_string() == "64" {
+            "42".to_string()
+        } else {
+            vm_req.vcpus.to_string()
+        };
```

Comment:
Wouldn't something like this be better:
```suggestion
        let vcpus = min(42, vm_req.vcpus).to_string()
```